src/Trigger/RabbitMQTriggerAttribute.cs

```diff
@@ -9,28 +9,31 @@ namespace Microsoft.Azure.WebJobs
     [Binding]
     public sealed class RabbitMQTriggerAttribute : Attribute
     {
-        public RabbitMQTriggerAttribute(string connectionStringSetting, string queueName, string deadLetterExchangeName = "")
+        public RabbitMQTriggerAttribute(string connectionStringSetting, string queueName, string deadLetterExchangeName = "", ushort batchNumber = 1)
         {
             ConnectionStringSetting = connectionStringSetting;
             QueueName = queueName;
             DeadLetterExchangeName = deadLetterExchangeName;
+            BatchNumber = batchNumber;
         }
 
 
-        public RabbitMQTriggerAttribute(string queueName, string deadLetterExchangeName = "")
+        public RabbitMQTriggerAttribute(string queueName, string deadLetterExchangeName = "", ushort batchNumber = 1)
         {
             QueueName = queueName;
             DeadLetterExchangeName = deadLetterExchangeName;
+            BatchNumber = batchNumber;
         }
 
-        public RabbitMQTriggerAttribute(string hostName, string userNameSetting, string passwordSetting, int port, string queueName, string deadLetterExchangeName = "")
+        public RabbitMQTriggerAttribute(string hostName, string userNameSetting, string passwordSetting, int port, string queueName, string deadLetterExchangeName = "", ushort batchNumber = 1)

```

Comment:
Why are user/password separate settings, given that they are part of the connection string?

Reply:
We wanted to give the user the option to pass in connection info either through a connection string or as separate parameters. That's why the constructor is there, but we haven't been using it much in our testing